cfme/tests/services/test_service_catalogs.py

```diff
@@ -16,7 +16,8 @@
 pytestmark = [
     pytest.mark.meta(server_roles="+automate"),
     pytest.mark.usefixtures('logged_in', 'vm_name', 'catalog_item', 'uses_infra_providers'),
-    pytest.mark.long_running
+    pytest.mark.long_running,
+    pytest.mark.tier(2)
```

Comment:
Tier 2 are appliance level smoke tests really, some of these tests look like actually they could be tier 2, but I don't think all of them.